### ISSUE TITLE
Face Recognition 

### ISSUE THREAD

**Issue by Shahriar-Shakir-Sumit** [tryolabs/luminoth#248]
After tracking person, can we track face from that data ?

**Comment by syedmustafa54** [comment]
Yes, You need to use a different algorithm called [FaceRecognition](https://github.com/ageitgey/face_recognition). Use luminoth for person extraction and the run [FaceRecognition](https://github.com/ageitgey/face_recognition) on that extracted person and please Close  this it is not an issue
